corehq/messaging/tasks.py

```diff
@@ -67,7 +67,7 @@ def _sync_case_for_messaging(domain, case_id):
     case_load_counter("messaging_sync", domain)()
     update_messaging_for_case(domain, case_id, case)
     if case is not None:
```

Comment:
In the pillow code this also checked if the case was deleted. Is that still necessary?

Reply:
I don't think so, both in the sense that this code has been running in production since before the pillow so in some sense defines "correct" behavior, but also in the sense that I don't think deleting a case can trigger a rule. So at worst this is probably a noop, although if it does trigger, it keeps this in line with existing behavior, so I would be reticent to change it.